test/utils/browser.js

```diff
@@ -81,6 +144,11 @@ exports.assertBrowser = function () {
   });
 };
 
+exports.simpleExecute = function (fn, args, cb) {
+  // DEV: Barebones execute function on the browser, this works when .execute does not
+  this.browser.execute(functionToString(fn).body, args, cb);
+};
```

Comment:
Yep, I got the signature correct when I was mucking around with execute, the thing is I'm pretty sure I copied and pasted a `.execute` from somewhere else that should have thrown an error or done something and nothing happened then either.  Could it be some funky `this` scoping issues